lib/graphql/language/lexer.rl

```diff
@@ -138,7 +138,7 @@ module GraphQL
       %% write data;
 
       def self.run_lexer(query_string)
-        data = query_string.unpack("c*")
```

Comment:
~~The generated output from this file has `frozen_string_literal: true`, so I would _expect_ this string to be reused~~

Nevermind, I see that this is already being used elsewhere for consistency, not performance :+1: